src/common/com/intellij/plugins/haxe/model/HaxeParameterModel.java

```diff
@@ -104,4 +109,20 @@ public String getPresentableText() {
     out += getType().toStringWithoutConstant();
     return out;
   }
+
+  public void remove() {
+    PsiElement psi = getPsi();
+    if (psi != null) {
+      PsiElement prePsi = UsefulPsiTreeUtil.getPrevSiblingSkipWhiteSpaces(psi, true);
+      PsiElement nextPsi = UsefulPsiTreeUtil.getNextSiblingNoSpaces(psi);
+      TextRange range = psi.getTextRange();
+
+      if (prePsi != null && prePsi.getText().equals(",")) {
+        range = range.union(prePsi.getTextRange());
+      } else if (nextPsi != null && nextPsi.getText().equals(",")) {
+        range = range.union(nextPsi.getTextRange());
```

Comment:
I have added a test that remove a parameter between two other parameters.
```a, ~~b~~, c```

I'm not sure if I understood right, but since it is an else if instead of an if, it would just remove one comma instead of two in the case there are commands, and would work fine.

I have added two more test cases (and fixed a small whitespace issue):
remove a parameter between two other parameters, and remove the first parameter.